index.js

```diff
@@ -2,6 +2,20 @@ const start = async () => {
 	// Set the desired es version for downstream modules that support it
 	(await import('@balena/es-version')).set('es2022');
 
+	const { NodeSDK } = await import('@opentelemetry/sdk-node');
+	const { ExpressInstrumentation } = await import(
+		'@opentelemetry/instrumentation-express'
+	);
+	const { HttpInstrumentation } = await import(
+		'@opentelemetry/instrumentation-http'
+	);
+
+	const sdk = new NodeSDK({
+		instrumentations: [new ExpressInstrumentation(), new HttpInstrumentation()],
+	});
+
+	sdk.start();
```

Comment:
The docs also mention a `shutdown()` that can be called to cleanly close the SDK client, but I suppose it's not required and we don't have an existing SIGTERM/etc handler here.